src/scss/patterns/_lists.scss

```diff
@@ -0,0 +1,58 @@
+@use '~accoutrement/sass/tools';
+@use '../config/scale';
+@use 'sass:map';
+
+// Lists
+// -----
+/// Apply spacing after lists.
+/// @group type
+ul,
+ol {
+  margin-bottom: var(--newline);
+  margin-top: 0;
+}
+
+ul,
+ol {
+  padding-left: var(--list-padding, var(--list-padding--default, 0));
+
+  & & {
+    --list-padding--default: 1em;
+  }
+
+  @include tools.below('wide-page') {
+    --list-padding--default: 1em;
+  }
+}
+
+// List Markers
+// ------------
+/// Lighter markers and smaller bullet points…
+/// @group type
+::marker {
+  color: var(--text-light);
```

Comment:
I would use a stack here, with something like `var(--markers, var(--text-light))`, and then have `[data-block='fancy-list']` below set `--markers: var(--icons);`

I think that's more accurately the pattern here. In general we want markers based on text, but sometimes we want them to look more like our icon styles. By using a stack, we can also avoid the deep nesting below. (Could do something similar with font-size, by adding a custom property like `--marker-size`, and falling back to `1em` here.